### PR TITLE
Handle drawing of the transport way on the map

### PR DESCRIPTION
**Pull request by lucasardonini**

### CODE REVIEW

File: mobile/src/main/java/ch/liip/timeforcoffee/activity/ConnectionsActivity.java [hunk 1]
```diff
@@ -200,6 +206,7 @@ public void performConnectionsUpdate() {
     }
 
     public void updateConnections(List<Connection> connections) {
+        mStationMapFragment.setup(connections, getResources().getString(R.string.connection_from));
```
Comment (FriedrichIII) [inline mobile/src/main/java/ch/liip/timeforcoffee/activity/ConnectionsActivity.java]: Resources are available in the fragment. Do you really need to pass them as parameter?

File: mobile/src/main/java/ch/liip/timeforcoffee/fragment/StationMapFragment.java [hunk 2]
```diff
@@ -12,147 +13,239 @@
...
-        });
-
-        Location currentLocation = SmartLocation.with(getActivity()).location().getLastLocation();
-        WalkingDistance distance = mStation.getDistanceForDisplay(currentLocation);
-        if (distance != null) {
-            subtitleTextView.setText(distance.getWalkingDistance());
         }
     }
 
     private void drawTransportPath() {
```
Comment (FriedrichIII) [inline mobile/src/main/java/ch/liip/timeforcoffee/fragment/StationMapFragment.java:190]: Maybe we could improve that by using Google direction. For now it displays straight line between checkpoint, which doesn't really match the path used by the transport line.

File: mobile/src/main/java/ch/liip/timeforcoffee/fragment/StationMapFragment.java [hunk 3]
```diff
@@ -12,147 +13,239 @@
...
+    private Callbacks mCallbacks = sDummyCallbacks;
+
+    public interface Callbacks {
+        void onMapLoaded();
     }
 
-    @Override
-    public void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
+    private static Callbacks sDummyCallbacks = new Callbacks() {
```
Comment (FriedrichIII) [inline mobile/src/main/java/ch/liip/timeforcoffee/fragment/StationMapFragment.java]: Smells like a workaround for some issue. Could you comment what it fixes and if it is really needed?

**Review by FriedrichIII — CHANGES_REQUESTED** [review]
Good starting implementation. Let's discuss IRL how to further improve it.

**Comment by lucasardonini** [comment]
@FriedrichIII Comments applied

**Review by FriedrichIII — APPROVED** [review]

File: mobile/src/main/java/ch/liip/timeforcoffee/fragment/StationMapFragment.java (hunk 2 above)
Comment (FriedrichIII) [reply mobile/src/main/java/ch/liip/timeforcoffee/fragment/StationMapFragment.java:190]: Created issue #65 to track this

**Pull request merged**